ci/templates/steps.yml

```diff
@@ -9,14 +9,17 @@ steps:
   - task: Cache@2
     inputs:
       key: '"questdb_main" | "$(Agent.OS)" | "$(jdk)" | "maven"'
-      restoreKeys: |
-        "questdb_main" | "$(Agent.OS)" | "$(jdk)" | "maven"
       path: $(MAVEN_CACHE_FOLDER)
+  - bash: |
+      echo "SOURCE_CODE_CHANGED: $(SOURCE_CODE_CHANGED)"
+      echo "RUST_SOURCE_CODE_CHANGED: $(RUST_SOURCE_CODE_CHANGED)"
+      echo "COVERAGE_DIFF: $(COVERAGE_DIFF)"
+      echo "CODE_COVERAGE_TOOL_OPTION: $(CODE_COVERAGE_TOOL_OPTION)"
+    displayName: "Print out task variables"
```

Comment:
nit: you don't need to do this, Azure job log these, you can find them if you expand "Job preparation parameters"


![image](https://github.com/user-attachments/assets/749a77c6-27cd-40dd-ac0e-d2578cce1303)

Reply:
Commit prepared that deletes this.